src/imitation/rewards/serialize.py

```diff
@@ -62,14 +100,36 @@ def f(
 
 
 # TODO(adam): I think we can get rid of this and have just one RewardNet.
+
 reward_registry.register(
     key="RewardNet_shaped",
-    value=_load_reward_net_as_fn(shaped=True),
+    value=lambda path, _: _validate_reward(_make_functional(th.load(str(path)))),
 )
 reward_registry.register(
```

Comment:
Seems to me `NormalizedRewardNet` and `ShapedRewardNet` are both subclasses of `RewardNetWrapper`. Could simplify things by registering a `"RewardNet_unwrapped"`.

Reply:
How should "RewardNet_unwrapped" behave in the presence of multiple wrappers? See https://github.com/HumanCompatibleAI/imitation/issues/437#issuecomment-1136575041.